src/identify/message.d.ts

```diff
@@ -34,25 +34,40 @@ export class Identify implements IIdentify {
     constructor(p?: IIdentify);
```

Comment:
all these changes to the random autogenerated protobuf code happened when I ran `npm run build:proto && npm run build:proto-types` on a totally clean checkout before I even started making any of my changes.

Reply:
If you want me to leave those changes out of this PR just let me know and I can unstage those files.

Reply:
Yes please!

Reply:
Yep will do.  I'm still worried though that the code generated for the new fetch protocol protobufs is going to have the same problems as clearly there's something in my local setup causing it to generate protobufs differently than what yall have here.